src/JUnit.Xml.TestLogger/JUnitXmlTestLogger.cs

```diff
@@ -241,10 +256,19 @@ public void Initialize(TestLoggerEvents events, Dictionary<string, string> param
         }
 
         /// <summary>
-        /// Called when a test message is received.
+        /// Called when a test message is received. These messages are coming from the test
+        /// framework, and don't contain standard output produced by test code.
         /// </summary>
         internal void TestMessageHandler(object sender, TestRunMessageEventArgs e)
```

Comment:
At a high level, I am wondering if we should add unit tests for the `internal` method of this class (atleast the ones we are changing). It will make our validation much easier.

Reply:
When we add support for parallel test runs, I think it would make sense. Right now, the behavior is so simple I think the tests cover it.

Reply:
Sounds good.